var/spack/repos/builtin/packages/ascent/package.py

```diff
@@ -169,13 +169,11 @@ class Ascent(CMakePackage, CudaPackage):
     # VTK-m
     #######################
 
-    depends_on("vtk-m@1.9:", when="@0.9.0:")
```

Comment:
@cyrush Is this always required now, or still only with vtkh and just missing the variant

Reply:
This would be needed only with `+vtkh` variant.